app/helpers/gtl_helper.rb

```diff
@@ -0,0 +1,125 @@
+module GtlHelper
+  def gtl_selected_records
+    records = params.try(:[], :rec_ids) || @edit.try(:[], :pol_items) || @edit.try(:[], :object_ids)
+
+    if records.present?
+      records = records.map(&:to_i) if records.first.kind_of?(String)
+      records = records.map(&:id) unless records.first.kind_of?(Integer)
+    end
```

Comment:
I would write something like that:
```ruby
def gtl_selected_records
  records = params.try(:[], :rec_ids) || @edit.try(:[], :pol_items) || @edit.try(:[], :object_ids)
  return [] if records.empty?

  case record.first
  when String
    records.map(&:to_i)
  when Integer
    records
  else
    records.map(&:id)
  end
end
```

Reply:
Well I would rather go and see that is the content. Of :rec_ids, :pol_items and :object_ids.

Most likely I will find out that each of the transformations is needed for a different source of data.

With that knowledge I can do a proper refactoring of this method.

The think is: Refactoring of this code IS NOT the topic of this PR. And I have other tasks to solve.